packages/run-protocol/src/runStake/runStake.js

```diff
@@ -0,0 +1,171 @@
+// @ts-check
+import { handleParamGovernance } from '@agoric/governance';
+import { E, Far } from '@endo/far';
+import { makeAttestationFacets } from './attestation.js';
+import { makeRunStakeParamManager } from './params.js';
+import { makeRunStakeKit } from './runStakeKit.js';
+import { makeRunStakeManager } from './runStakeManager';
+
+/**
+ * Provide loans on the basis of staked assets that earn rewards.
+ *
+ * In addition to brands and issuers for `Staked`, `RUN`, and attestation,
+ * terms of the contract include a periodic interest rate plus
```

Comment:
would help to have the term keywords mentioned in the text.
```suggestion
 * terms of the contract include a periodic interest rate (`InterestRate`) plus
```